docker/local/Dockerfile

```diff
@@ -14,7 +14,51 @@
 # - Allow persisting settings on disk
 # - Limiting persisted data with some auto rotation
 
-FROM clickhouse/clickhouse-server:23.11.1-alpine AS base
+# Get Node base image to copy over Node binaries
+ARG NODE_VERSION=18.15.0
+FROM node:${NODE_VERSION}-alpine AS node
+
+# == API Builder Image ==
+FROM node:${NODE_VERSION}-alpine AS api_builder
+
+WORKDIR /app/api
+COPY ./yarn.lock .
+COPY --from=api ./package.json .
+RUN yarn install --frozen-lockfile && yarn cache clean
+
+COPY --from=api ./tsconfig.json .
+COPY --from=api ./src ./src
+RUN yarn run build
+
+# == App Builder Image ==
+FROM node:${NODE_VERSION}-alpine AS app_builder
+
+WORKDIR /app/app
+
+COPY ./yarn.lock ./.yarnrc .
+COPY ./.yarn ./.yarn
+COPY --from=app ./package.json .
+
+RUN yarn install --frozen-lockfile && yarn cache clean
+
+COPY --from=app ./.eslintrc.js ./next.config.js ./tsconfig.json ./next.config.js ./mdx.d.ts ./.eslintrc.js ./
+COPY --from=app ./src ./src 
+COPY --from=app ./pages ./pages 
+COPY --from=app ./public ./public 
+COPY --from=app ./styles ./styles
+
+ENV NEXT_TELEMETRY_DISABLED 1
+ENV NEXT_OUTPUT_STANDALONE true
+RUN yarn build
+
+# == Clickhouse/Base Image ==
+
+FROM clickhouse/clickhouse-server:23.11.1-alpine AS clickhouse_base 
```

Comment:
I would suggest to align the ch version with the prebuilt and dev envs (v23.8.8)